django_project/base/templates/project/new_detail.html

```diff
@@ -98,7 +98,12 @@ <h4 class="text-muted">
                     <div class="version-list">
                     {% for version in versions %}
                         {% if version.approved %}
-                        <p><span class="text-muted">Version:</span> <span class="pull-right">{{ version.name }}</span></p>
+                        <p><span class="text-muted">Version:</span>
```

Comment:
May be put one tag in one line will make it's more readable:
```
<p>
    <span>

    </span>
</p>
```

Reply:
Thanks @ismailsunni 
I'll do so.